src/foam/nanos/auth/resetPassword/ResendView.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth.resetPassword',
+  name: 'ResendView',
+  extends: 'foam.u2.View',
```

Comment:
If you're exporting 'as data' below you can just make this extend foam.u2.Controller instead and then remove the exports:.